channeldb/forwarding_log.go

```diff
@@ -271,27 +274,31 @@ func (f *ForwardingLog) Query(q ForwardingEventQuery) (ForwardingLogTimeSlice, e
 				continue
 			}
 
-			currentTime := time.Unix(
-				0, int64(byteOrder.Uint64(timestamp)),
-			)
-
 			// At this point, we've skipped enough records to start
 			// to collate our query. For each record, we'll
 			// increment the final record offset so the querier can
 			// utilize pagination to seek further.
-			readBuf := bytes.NewReader(events)
-			for readBuf.Len() != 0 {
-				var event ForwardingEvent
-				err := decodeForwardingEvent(readBuf, &event)
-				if err != nil {
-					return err
-				}
-
-				event.Timestamp = currentTime
-				resp.ForwardingEvents = append(resp.ForwardingEvents, event)
-
-				recordOffset++
+			readBuf := bytes.NewReader(eventBytes)
+			if readBuf.Len() == 0 {
+				continue
 			}
+
+			currentTime := time.Unix(
```

Comment:
Nit: maybe we can rename it to eventTime, current time seems a bit misleading ?

Reply:
sure can do!